src/scene/materials/standard-material.js

```diff
@@ -262,6 +262,8 @@ import { standardMaterialCubemapParameters, standardMaterialTextureParameters }
  * * occludeSpecularFloat: defines if {@link pc.StandardMaterial#occludeSpecularIntensity} constant should affect specular occlusion.
  * * alphaTest: enable alpha testing. See {@link pc.Material#alphaTest}.
  * * alphaToCoverage: enable alpha to coverage. See {@link pc.Material#alphaToCoverage}.
+ * * opacityFadesSpecRefl: used to specify whether specular and reflections are faded out using opacity. Default is true. When set to false use alphaFade to fade out materials.
+ * * alphaFade: used to fade out materials when opacityFadesSpecRefl is set to false.
```

Comment:
These properties are documented as inputs for `onUpdateShader` - but are they not just public on the `StandardMaterial` itself?

Reply:
Also, I'm not really understanding this `alphaFade` property. Is this equivalent to `opacity` and used instead of `opacity` when `opacityFadesSpecRefl` is false?

Reply:
They should be public - I will add them

Reply:
done - thanks!